src/app/containers/ChartbeatAnalytics/canonical/index.jsx

```diff
@@ -29,7 +29,7 @@ const CanonicalChartbeatBeacon = ({ chartbeatConfig }) => {
         })();
       `}
       </script>
-      <script async type="text/javascript" src={chartbeatSource} />
+      <script defer type="text/javascript" src={chartbeatSource} />
```

Comment:
Just to clarify, with this change Chartbeat won't load until the full document has been parsed?

Reply:
With `async`, we download the script in parallel to parsing the page, and execute it as soon as it is available.

With `defer`, we download the script in parallel to parsing the page, but don't execute it until after parsing is finished.